tensorflow_probability/python/bijectors/real_nvp.py

```diff
@@ -105,7 +105,7 @@ class RealNVP(bijector_lib.Bijector):
 
   x = nvp.sample()
   nvp.log_prob(x)
-  nvp.log_prob(0.)
+  nvp.log_prob((0.0, 0.0, 0.0))
```

Comment:
```
ipdb> nvp.log_prob((0.0, 0.0, 0.0))
<tf.Tensor: shape=(), dtype=float32, numpy=-2.7568154>
```

Reply:
Would you mind changing this to be a list rather than a tuple? We (as a style point) prefer to use lists whenever possible for arrays of numbers.

Reply:
Changed. Thanks!